code/__DEFINES/~monkestation/jobs.dm

```diff
@@ -1 +1,2 @@
 #define JOB_SECURITY_ASSISTANT "Security Assistant"
+#define JOB_BLUESHIELD "Blueshield"
```

Comment:
Unsure if this is needed since you're already defining it in main code?

Reply:
I actually don't think I did, should have been the only time I did it